pkg/consts/consts.go

```diff
@@ -41,6 +41,8 @@ const (
 	MinSquareSize = 1
 	// MinshareCount is the minimum shares required in an original data square.
 	MinSharecount = MinSquareSize * MinSquareSize
+
+	TxInclusionProofQueryPath = "custom/txInclusionProof/%d"
```

Comment:
[optional]

```suggestion
	// TxInclusionProofQueryPath is the HTTP API path that celestia-app exposes
	// to handle producing transaction inclusion proofs. %d is expected to be
	// substituted with the transaction index.
	TxInclusionProofQueryPath = "custom/txInclusionProof/%d"
```

Reply:
used different wording to reflect that it uses the ABCI not http
4e9cf0b11bfa13e1b1a3bf2a9da890884962e1d2
bfb765a1ad1cf5f2c58517bbbc1a164447eb0558


there are no servers involved, and our version of ABCI++ only supports in process applications

Reply:
[question] does this need the `custom/` prefix? If so, does the celestia-app version need it?

https://github.com/celestiaorg/celestia-app/pull/708#discussion_r971262154

Reply:
this one does, the celestia-app one shouldn't tho